### PR TITLE
[incubator/mattermost] Initial commit of Mattermost chart.

### PR DESCRIPTION
**Pull request by jpds**

### CODE REVIEW

File: incubator/mattermost/templates/pvc.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{{- if and .Values.persistence.enabled -}}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}-app
+  labels:
+    app: {{ template "fullname" . }}
```
Comment (unguiculus) [inline incubator/mattermost/templates/pvc.yaml]: `app` label should be  `{{ template "name" . }}`.

File: incubator/mattermost/templates/secrets.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
```
Comment (unguiculus) [inline incubator/mattermost/templates/secrets.yaml]: `app` label should be  `{{ template "name" . }}`.

File: incubator/mattermost/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
+name: mattermost
+version: 0.1.0
```
Comment (unguiculus) [inline incubator/mattermost/Chart.yaml]: `Chart.yaml` incomplete. Please add `appVersion`, `icon`, `home`, `sources`, `maintainers`, etc.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR. Looks pretty good.

**Comment by jpds** [comment]
Fixes pushed, thanks.

File: incubator/mattermost/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+description: A Mattermost Helm chart for Kubernetes
+name: mattermost
+version: 0.1.0
+appVersion: 3.10.0
+home: https://about.mattermost.com/
+icon: https://www.mattermost.org/wp-content/uploads/2016/04/icon.png
+sources:
+  - https://github.com/mattermost/platform
+maintainers:
+  - name: Jonathan Davies
```
Comment (unguiculus) [inline incubator/mattermost/Chart.yaml]: Please use your Github handle as name.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: incubator/mattermost/README.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+| `image.repository`      | Container image to use              | `mattermost/mattermost-prod-app`                    |
+| `image.tag`             | Container image tag to deploy       | `3.10.0`                                            |
+| `mattermost.db.host`    | Database host to use for Mattermost | `db`                                                |
+| `mattermost.db.port`    | Database port to use for Mattermost | `5432`                                              |
+| `mattermost.db.name`    | Database name to use for Mattermost | `mattermost`                                        |
+| `mattermost.db.username`| Database username to use for Mattermost | `mmuser`                                        |
+| `mattermost.db.password`| Database password to use for Mattermost | `mmpassword`                                    |
+| `replicaCount`          | k8s replicas                        | `1`                                                 |
+| `resources.limits.cpu`  | Container requested CPU             | `nil`                                               |
```
Comment (unguiculus) [inline incubator/mattermost/README.md:44]: You should just specify `resources` with default `{}`.

File: incubator/mattermost/values.yaml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    - chart-example.local
+  annotations:
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  tls:
+    # Secrets must be manually created in the namespace.
+    # - secretName: chart-example-tls
+    #   hosts:
+    #     - chart-example.local
+mattermost:
```
Comment (unguiculus) [inline incubator/mattermost/values.yaml:26]: Why is just the DB under `mattermost`? You could probably put everything under that so I'd just remove it.

**Review by unguiculus — CHANGES_REQUESTED** [review]
A few more things I noticed.

**Comment by unguiculus** [comment]
Marking as stale. Please update within the next week.

**Comment by mccricardo** [comment]
Any progress on this?

**Comment by unguiculus** [comment]
Closing as stale. Feel free to reopen if/when you can work on this again.

**Pull request closed without merging**